lib/taurus/core/tango/tangoattribute.py

```diff
@@ -811,7 +811,11 @@ def setRange(self, *limits):
             limits = limits[0]
         low, high = limits
         low = Quantity(low)
+        if low.dimensionless:
+            low = Quantity(low, self._units)
```

Comment:
>  I will try with the docker.

A priori it works, but when I ask for the version pint returns "'unknown-taurus'"